hub/saml-proxy/src/test/resources/saml-proxy.yml

```diff
@@ -95,25 +95,27 @@ metadata:
       verifyHostname: false
       trustSelfSignedCertificates: true
 
-eidas: true
-countryMetadata:
```

Comment:
Good thing `ignoreUnknownProperties` was on and so this was completely ignored 😱